src/styles/block/contact.scss

```diff
@@ -0,0 +1,113 @@
+.contact {
+  margin-top: 80px;
+
+  @include OnTablet {
+    margin-top: 120px;
+  }
+
+  @include OnDesktop {
+    margin-top: 180px;
+  }
+
+  &__wrapper {
+    @include PageGrid;
+    row-gap: 48px;
+  }
+
+  &__rows {
+    grid-column: 1/-1;
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+
+    @include OnTablet {
+      grid-column: span 3;
+    }
+
+    @include OnDesktop {
+      grid-column: span 5;
+    }
+
+    &--input {
+      box-sizing: border-box;
```

Comment:
box-sizing: border-box should be applied to every element
* { 
 box-sizing: border-box;
}

Reply:
If add to all doc, header is wrong position

Reply:
![Uploading image.png…]()

Reply:
![image](https://github.com/mate-academy/layout_landing-page/assets/26773495/f8767799-5521-4c2f-b9d9-df7204c4dd5b)